discovery-provider/src/schemas/track_schema.json

```diff
@@ -81,12 +82,14 @@
                 "field_visibility": {
                     "type": "object",
                     "$ref": "#/definitions/FieldVisibility",
-                    "default": null
+                    "default": false,
```

Comment:
shouldn't this default be null instead of false?

Reply:
good catch, did not mean to do this. fixed